src/order/locally_finite.lean

```diff
@@ -482,6 +482,255 @@ locally_finite_order.of_Icc' (α × β)
 
 end preorder
 
+/-! #### `with_top`, `with_bot` -/
+
+section order_top
+open with_top
+variables (α) [decidable_eq α] [order_top α] [locally_finite_order α]
```

Comment:
Do you really want `order_top` here? Adding a top to a type already equipped with one seems strange